core-blocks/shortcode/index.php

```diff
@@ -0,0 +1,29 @@
+<?php
+/**
+ * Server-side rendering of the `core/shortcode` block.
+ *
+ * @package gutenberg
+ */
+
+/**
+ * Performs wpautop() on the shortcode block content.
```

Comment:
Could be improved to explain _why_ we're `autop`-ing, not simply expressing that we are, which is already evident in the code itself.

Alternatively, it could be considered an implementation detail that the block's render applies autop, not necessarily appropriate for the callback function. In other words, this function should be documented along lines of "Renders the shortcode", and the implementation detail explained by an inline code comment.